src/components/MultiHopTrade/MultiHopTrade.tsx

```diff
@@ -49,54 +48,39 @@ const MultiHopEntries = [
 
 export type TradeCardProps = {
   defaultBuyAssetId?: AssetId
-  defaultSellAssetId?: AssetId
 } & CardProps
 
 type MatchParams = {
   chainId?: string
   assetSubId?: string
 }
 
-export const MultiHopTrade = memo(
-  ({
-    defaultBuyAssetId: _defaultBuyAssetId = foxAssetId,
-    defaultSellAssetId = ethAssetId,
-    ...cardProps
-  }: TradeCardProps) => {
-    // Overrides the defaultBuyAssetId to BTC across all instances of this component, to avoid all consumers programmatically checking for isSnapInstalled
-    const isSnapInstalled = useIsSnapInstalled()
-    const defaultBuyAssetId =
-      _defaultBuyAssetId === foxAssetId && isSnapInstalled ? btcAssetId : _defaultBuyAssetId
-    const dispatch = useAppDispatch()
-    const methods = useForm({ mode: 'onChange' })
-    const { assetSubId, chainId } = useParams<MatchParams>()
+export const MultiHopTrade = memo(({ defaultBuyAssetId, ...cardProps }: TradeCardProps) => {
+  const dispatch = useAppDispatch()
+  const methods = useForm({ mode: 'onChange' })
+  const { assetSubId, chainId } = useParams<MatchParams>()
 
-    const defaultBuyAsset = useAppSelector(state => selectAssetById(state, defaultBuyAssetId))
-    const routeBuyAsset = useAppSelector(state =>
-      selectAssetById(state, `${chainId}/${assetSubId}`),
-    )
-    const defaultSellAsset = useAppSelector(state => selectAssetById(state, defaultSellAssetId))
+  const routeBuyAsset = useAppSelector(state => selectAssetById(state, `${chainId}/${assetSubId}`))
+  const defaultBuyAsset = useAppSelector(state => selectAssetById(state, defaultBuyAssetId ?? ''))
 
-    useEffect(() => {
-      dispatch(tradeInput.actions.clear())
-      if (defaultSellAsset) dispatch(tradeInput.actions.setSellAsset(defaultSellAsset))
-      if (routeBuyAsset) dispatch(tradeInput.actions.setBuyAsset(routeBuyAsset))
-      else if (defaultBuyAsset) dispatch(tradeInput.actions.setBuyAsset(defaultBuyAsset))
-    }, [defaultBuyAsset, defaultSellAsset, dispatch, routeBuyAsset])
+  useEffect(() => {
+    dispatch(tradeInput.actions.clear())
```

Comment:
Is the removed logic for the sell asset setting on purpose here?

Reply:
yes it is, the store has a default asset for both buy and sell and only the default buy asset is overridden in this case